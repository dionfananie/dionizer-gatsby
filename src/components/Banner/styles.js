import { css } from 'emotion';

export const titleBanner = css`
  font-family: 'Courgette', cursive;
`;
