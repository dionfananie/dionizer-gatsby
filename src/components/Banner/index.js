import React from 'react';
import { titleBanner, wrapperBanner } from './styles';
import workImg from '@images/work.svg';
import propTypes from 'prop-types';

const Banner = ({ title }) => {
  return (
    <div className={wrapperBanner}>
      <img src={workImg} width="350" alt="" />
      <h1 className={titleBanner}>{title}</h1>
    </div>
  );
};

Banner.propTypes = {
  title: propTypes.string,
};

Banner.defaultProps = {
  title: ``,
};

export default Banner;
