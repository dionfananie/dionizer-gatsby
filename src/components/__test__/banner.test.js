import React from 'react';
import renderer from 'react-test-renderer';
import Banner from '../Banner';

describe('Banner', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Banner title="Collaborate" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
