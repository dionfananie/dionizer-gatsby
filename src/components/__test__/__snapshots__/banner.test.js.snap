// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner renders correctly 1`] = `
<div
  className="css-gxfm47"
>
  <img
    alt=""
    src="test-file-stub"
    width="350"
  />
  <h1
    className="css-1cf31kz"
  >
    Collaborate
  </h1>
</div>
`;
